tests.py

```diff
@@ -23,27 +23,27 @@ def test_trasportation_for_non_square_matrix():
     assert expected == trasportation(general_matrix)
 
 def test_gauss_test1():
-    expected = to_fractions([[1, 3, 1, -1], [0.0, -5.0, 3.0, 4.0], [0.0, 0.0, 1.0, 4.0], [0.0, 0.0, 0.0, Fraction(7,5)]])
+    expected = to_fractions([[1, 3, 1, -1], [0, -5, 3, 4], [0, 0, 1, 4], [0, 0, 0, Fraction(7,5)]])
     switch,new_matrix = gauss(square_matrix)
     assert pow(-1,switch) == -1 # controllo se switch è pari
     assert expected == new_matrix
     
 def test_sum_line():
-    line1 = [1,2,3,4]
-    line2 = [-1,-2,-3,-4]
+    line1 = line_to_fractions([1,2,3,4])
+    line2 = line_to_fractions([-1,-2,-3,-4])
     sum_line(line1,line2)
-    assert [0,0,0,0] == line1
-    line1 = [3,6,-1,6]
-    line2 = [0.5,-0.5,1,-3]
+    assert line_to_fractions([0,0,0,0]) == line1
+    line1 = line_to_fractions([3,6,-1,6])
+    line2 = line_to_fractions([Fraction(1,2),Fraction(-1,2),1,-3])
```

Comment:
lol si mettono da qua i commenti mi sa 
comunque perchè dentro 
```
    line2 = line_to_fractions([Fraction(1,2),Fraction(-1,2),1,-3])
```
li chiami con `Fractions(1,2)` invece di `'1/2'` tanto poi line to fractions la ripete di nuovo 
poi se scritte come stringhe sono anche più leggibili nei test